packages/mendel-pipeline/src/client/generators.js

```diff
@@ -60,6 +68,12 @@ class MendelGenerators {
             this.perform(bundle, doneBundles);
         });
 
+        // If printEntryCount is set, print number of entries collected
+        // by each generator.
+        if (this.printEntryCount) {
```

Comment:
`verbose` is not passed to `generator`. BaseClient sends only `config` to `generator`.